lib/internal/errors.js

```diff
@@ -56,7 +57,12 @@ class SystemError extends Error {
     if (context.dest !== undefined)
       message += ` => ${context.dest}`;
 
-    super(message);
```

Comment:
Why this change? I'd prefer to have it there.

Reply:
I prefer it that way as well. The problem is that you may not use `this` before calling `super()` and we need the context to be able to attach properties to the error from within the function that constructs the error message.